modules/client-configuration/pages/autoinst-owngpgkey.adoc

```diff
@@ -40,6 +39,12 @@ You must add our own key to this file, so it can be used to check the packages.
 To access a shell inside the Server container run [literal]``mgrctl term`` on the container host.
 ====
 
+[IMPORTANT]
+====
+To copy GPG keys to the container use [command]``mgradm gpg add``.
+====
+
+
 .Procedure: Add a GPG key to the initial RAM disk
```

Comment:
This part can be removed. The user just need to run `mgradm gpg add` then `mgradm restart` .
- `mgradm gpg add` command copies the GPG keys and add it to the customer keyring 
- `mgradm restart` create a unique keyring using the suse and the customer keyring

All the application should use the unique keyring mentioned ^^.

Reply:
Thanks, Implemented. Please see the attached PDF.